float64 max_alt  # Maximum altitude in meters, w.r.t sea level.
float64 min_alt  # Minimum altitude in meters, w.r.t sea level.
GeoPolygon zone  # Polygon with points in WGS84 coordinates.
